nixos/modules/system/boot/binfmt.nix

```diff
@@ -331,6 +331,7 @@ in {
           "proc-sys-fs-binfmt_misc.mount"
           "systemd-binfmt.service"
         ];
+        services.systemd-binfmt.after = [ "systemd-tmpfiles-setup.service" "systemd-tmpfiles-resetup.service" ];
```

Comment:
Not fully sure if the `-resetup.service` is needed here? I assume it is so that we can support restarting `systemd-binfmt` in `switch-to-configuration` ?

Reply:
Shouldn't be needed. When restarted by `switch-to-configuration`, that will happen after it has already restarted `sysinit-reactivation`

Reply:
But I guess I do need to add a dependency on `sysinit-reactivation` to `systemd-binfmt` then?

Reply:
No. It's a separate transaction started by stc

Reply:
Gotcha